<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LazyDog</class>
 <widget class="QMainWindow" name="LazyDog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>LazyDogTools</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTextBrowser" name="debugBrowser">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_process">
       <attribute name="title">
        <string>已打开进程</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="5">
         <widget class="QTableView" name="tableView_process"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBox_add"/>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_renew">
          <property name="text">
           <string> 刷新列表</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="pushButton_add">
          <property name="text">
           <string>添加绑定</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_monitor">
       <attribute name="title">
        <string>已绑定进程</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="4" colspan="2">
         <widget class="QPushButton" name="pushButton_change">
          <property name="text">
           <string>更改绑定</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="8">
         <widget class="QTableView" name="tableView_monitor"/>
        </item>
        <item row="1" column="6" colspan="2">
         <widget class="QPushButton" name="pushButton_delete">
          <property name="text">
           <string>取消绑定</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="comboBox_change"/>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_seting">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="8" column="0" colspan="7">
         <widget class="QGroupBox" name="groupBox_mode">
          <property name="title">
           <string>模式</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QRadioButton" name="radioButton_window">
             <property name="text">
              <string>窗口模式</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButton_process">
             <property name="text">
              <string>进程模式</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="toolButton_modehelp">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;进程模式：根据&lt;span style=&quot; color:#aa0000;&quot;&gt;任务的启动顺序&lt;/span&gt;进行切换设备&lt;/p&gt;&lt;p&gt;窗口模式：根据&lt;span style=&quot; color:#aa0000;&quot;&gt;当前活动的窗口&lt;/span&gt;进行切换设备&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="15" column="6">
         <widget class="QPushButton" name="pushButton_about">
          <property name="text">
           <string>关于</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="7">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="15" column="0">
         <widget class="QPushButton" name="pushButton_checknew">
          <property name="text">
           <string>检查更新</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="7">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="15" column="1" colspan="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0" colspan="7">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="7">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="7">
         <widget class="QGroupBox" name="groupBox_keys">
          <property name="title">
           <string>快捷键</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>切换上一个设备</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>切换模式</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QKeySequenceEdit" name="keySequenceEdit_start">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>开启/关闭功能</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>切换下一个设备</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QKeySequenceEdit" name="keySequenceEdit_changemode">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QKeySequenceEdit" name="keySequenceEdit_changeup">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QKeySequenceEdit" name="keySequenceEdit_changedown">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2" colspan="7">
         <widget class="QGroupBox" name="groupBox_default">
          <property name="title">
           <string>首选项</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0" colspan="5">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="checkBox_autostart">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>开机自启动</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_adminstart">
                <property name="text">
                 <string>管理员模式启动</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="checkBox_defaultdevice">
             <property name="text">
              <string>默认设备</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QToolButton" name="toolButton_defaulthelp">
             <property name="toolTip">
              <string>当没有匹配到任何绑定的进程时，切回到默认设备</string>
             </property>
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="comboBox_defaultdevice"/>
           </item>
           <item row="7" column="3" colspan="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" colspan="5">
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="checkBox_autohidden">
                <property name="text">
                 <string>启动自动隐藏</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_changemsg">
                <property name="text">
                 <string>切换时通知</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="6" column="0" colspan="5">
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="checkBox_autochecknew">
                <property name="text">
                 <string>自动更新</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
